### PR TITLE
Fix bug with XDMF export in mode last 

### PR DESCRIPTION
**Pull request by RemDelaporteMathurin**
## Proposed changes

This is the fix for #738 

## Types of changes

What types of changes does your code introduce to FESTIM?

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [ ] Documentation Update (if none of the other choices apply)
- [ ] New tests

## Checklist

- [x] Black formatted
- [x] Unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: test/system/test_misc.py [hunk 1]
```diff
@@ -546,3 +546,56 @@ def test_error_raised_when_no_IC_heat_transfer():
         match="Initial condition is required for transient heat transfer simulations",
     ):
         my_model.initialise()
+
+
+def test_catch_bug_738(tmpdir):
+    """
+    Test to catch bug #738
+    Set up a simple simulation with and XDMFExport in write mode "last"
```
Comment (KulaginVladimir) [inline test/system/test_misc.py:554]: Maybe, just `with` or `and`?

**Review by KulaginVladimir — CHANGES_REQUESTED** [review]
@RemDelaporteMathurin, fix works fine. Just a comment on the test description.

File: test/system/test_misc.py (hunk 1 above)
Comment (RemDelaporteMathurin) [reply test/system/test_misc.py]: Good catch, this was a typo and should've been "with an XDMFExport"

Comment (KulaginVladimir) [reply test/system/test_misc.py]: Will you merge this PR?

Comment (RemDelaporteMathurin) [reply test/system/test_misc.py]: Yep unless you have other comments?

Comment (KulaginVladimir) [reply test/system/test_misc.py]: Nope

**Pull request merged**